modules/ducktests/tests/ignitetest/tests/two_phased_rebalanced_test.py

```
@@ -0,0 +1,294 @@
+        assert not diff, f"Validation error, files are different. Difference:\n {diff}"
+
+    def start_cells(self, config: IgniteConfiguration) -> List[IgniteService]:
+        """
+        Start cells.
+        :param config IgniteConfiguration.
+        :return List of IgniteServices.
+        """
+
+        cells = []
+
+        cell = start_cell(self.test_context, config, NUM_NODES_CELL, [f'-D{ATTRIBUTE}=0'])
+
+        discovery_spi = from_ignite_cluster(cell)
+        config = config._replace(discovery_spi=discovery_spi)
+
+        cells.append(cell)
+
+        for i in range(1, NUM_CELL):
+            cells.append(start_cell(self.test_context, config, NUM_NODES_CELL, [f'-D{ATTRIBUTE}={i}']))
+
+        return cells
+
+    def get_pds_size(self, cells: [IgniteService], msg: str) -> dict:
+        """
+        Pds size in megabytes.
+        :param cells List of IgniteService.
+        :param msg Information message.
+        :return dict with hostname -> pds size in megabytes.
+        """
+        assert len(cells) > 0
+
+        res = {}
+        for cell in cells:
+            for node in cell.nodes:
+                consistent_id = str(node.account.hostname).replace('.', '_').replace('-', '_')
+                cmd = f'du -sm {cell.database_dir}/{consistent_id} | ' + "awk '{print $1}'"
+                res[node.account.hostname] = int(node.account.ssh_output(cmd).decode("utf-8").rstrip())
+
+        self.logger.info(msg)
+
+        for item in res.items():
+            self.logger.info(f'Host: {item[0]}, PDS {item[1]}mb')
+
+        return res
+
+    def await_cluster_idle(self, node: ClusterNode):
+        """
+        Await Skipping checkpoint.
+        :param node ClusterNode.
+        """
+        try:
+            IgniteAwareService.await_event_on_node('Skipping checkpoint', node, timeout_sec=60)
+        except errors.TimeoutError as ex:
+            self.logger.warn(ex)
```

Why do you ignore this error?

---

it may not happen after start waiting